examples/rac-spectrum-tailwind/package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "rac-spectrum-tailwind-example",
   "private": true,
+  "packageManager": "yarn@4.2.2",
   "scripts": {
     "start": "parcel src/index.html",
     "build": "parcel build src/index.html",
     "install-17": "yarn add -W react@^17 react-dom@^17"
   },
   "dependencies": {
     "@adobe/react-spectrum": "^3.28.0",
+    "@react-spectrum/provider": "^3.9.7",
```

Comment:
where did this direct dependency come from?

Reply:
I noticed a couple packages had a specific minimum range, so I figured this was a good idea to just always update like the rest of our dependencies